terraform/step4/REDIS.tf

```diff
@@ -0,0 +1,28 @@
+# Redis subnet group
+resource "aws_elasticache_subnet_group" "redis_subnet_grp" {
+  name        = "${var.project_name}-redis-subnet-grp"
+  description = "Subnet group for redis databases"
+  subnet_ids  = var.vpc_private_subnets[*]
+  tags = {
+    Name = "${var.project_name}-redis-subnet-grp"
+  }
+}
+
+
+# Redis-main
+resource "aws_elasticache_cluster" "redis-main" {
```

Comment:
What are we using Redis for in DroneTM?

Reply:
@spwoodcock Initially i thought somewhere we would use celery for background task so i added configuration for Redis, For now i will comment this in the code and make a new PR.